tlatools/org.lamport.tlatools/src/tlc2/tool/liveness/TableauNodePtrTable.java

```diff
@@ -514,7 +535,7 @@ public static String toString(final TableauNodePtrTable table) {
   						// element
   						final long elem = getElem(node, j);
   						if (AbstractDiskGraph.isFilePointer(elem)) {
-  							if (table.isDone(fp)) {
+  							if (elem != UNDONE) {
   								buf.append("  ptr: " + elem);
   							} else {

```

Comment:
Should this also have a branch for the `DONE` case?

(It looks like this is just for debugging, so perhaps it isn't terribly important.)

Reply:
I'll leave this one out to make the bugfix commit cleanly apply against the 1.7.x branch.